ChangeLog

```diff
@@ -1,4 +1,12 @@
-+ Opencryptoki 3.14
++ openCryptoki 3.15
+- common: conform to PKCS 11 3.0 Baseline Provider profile
+- CCA: support key wrapping
+- EP11: support m_ReEncryptSingle
+- SOFT: support ECC
+- p11sak tool: remove key command
```

Comment:
a very minor comment: I would change "remove key command" to "add remove-key command"

Reply:
good point. done.